telemetry-core/src/main/java/com/newrelic/telemetry/metrics/MetricBatchSender.java

```diff
@@ -106,6 +120,27 @@ public static MetricBatchSender create(SenderConfiguration configuration) {
     return new MetricBatchSender(marshaller, sender);
   }
 
+  public static URL returnEndpoint(String userRegion) throws MalformedURLException {
+    URL url = null;
+    if (userRegion.equals("US")) {
+      try {
+        url = new URL(DEFAULT_URL.substring(0, DEFAULT_URL.length() - 1) + METRICS_PATH);
+        return url;
+      } catch (MalformedURLException e) {
+        e.printStackTrace();
+      }
+    } else if (userRegion.equals("EU")) {
+      try {
+        url = new URL(EUROPEAN_URL + METRICS_PATH);
```

Comment:
Can we make the creation of the Default URL be simple and consistent with the creation of the European Url?